components/blocks/Hero.template.ts

```diff
@@ -78,6 +78,12 @@ export const heroTemplate: TinaTemplate = {
       label: 'Custom Spacing',
       type: 'string',
     },
+    {
+      name: 'mobileTextSize',
+      description: 'Default text size on mobile is xl',
+      label: 'Mobile Text Size (toggle for smaller body text)',
+      type: 'boolean',
+    },
```

Comment:
I found this field confusing

```suggestion
    {
      name: 'smallerBodyText',
      description: 'Default text size on mobile is xl',
      label: 'Mobile Text Size (off = xl, on = lg)',
      type: 'boolean',
    },
```